roles/contiv_network/tasks/main.yml

```diff
@@ -20,21 +20,35 @@
 - name: download netmaster and netplugin
   get_url:
     validate_certs: "{{ validate_certs }}"
-    url: https://github.com/contiv/netplugin/releases/download/v0.0.0-12-11-2015.20-54-40.UTC/netplugin-v0.0.0-12-11-2015.20-54-40.UTC.tar.bz2
+    url: https://github.com/contiv/netplugin/releases/download/v0.1-01-28-2016.03-55-05.UTC/netplugin-v0.1-01-28-2016.03-55-05.UTC.tar.bz2
```

Comment:
@jainvipin: Trying to understand it better here. If the default is -docker-plugin, it should not error out even if the argument was provided right? I see that netplugin errors out when provided with this option.